src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -201,12 +201,40 @@ private static OpenApiResponses GetOpenApiResponses(MethodInfo method, EndpointM
                 responseContent[contentType] = new OpenApiMediaType();
             }
 
-            responses[statusCode] = new OpenApiResponse { Content = responseContent };
+            responses[statusCode] = new OpenApiResponse
+            {
+                Content = responseContent,
+                Description = GetResponseDescription(statusCode)
+            };
         }
-
         return responses;
     }
 
+    private static string GetResponseDescription(string statusCode)
+    {
+        if (statusCode.Length != 3)
+        {
+            return "";
```

Comment:
```suggestion
            return string.Empty;
```

This is a property that represents a 0-length string. ([ref](https://docs.microsoft.com/en-us/dotnet/api/system.string.empty?view=net-6.0))